lib/msal-browser/src/network/FetchClient.ts

```diff
@@ -27,6 +27,7 @@ export class FetchClient implements INetworkModule {
      */
     async sendGetRequestAsync<T>(
         url: string,
+        correlationId: string,
```

Comment:
Why wasn't this added as an optional param instead of mandated changing the interface? We can accommodate any changed in v5 as needed. We expose INetworkModule as a customizable interface to users, wouldn't this change it?